Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -27,6 +23,7 @@ public class ButtonRenderer : ViewRenderer<Button, AppCompatButton>, AView.IOnAt
 
 		public ButtonRenderer()
 		{
+			System.Diagnostics.Debug.WriteLine("Slow Button!");
```

Comment:
This is something we also added on non app compact button so users select fast renderers, should we remove it @ez ?